### PR TITLE
Add tutorial for tabular data classification

### PR DESCRIPTION
**Pull request by weijinglok**
This PR introduces the tabular data classification tutorial to the docs site. 

This tutorial has been reviewed by @jwmueller, but may benefit if reviewed by other reviewers too.

### CODE REVIEW

File: docs/source/notebooks/tabular_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab.classification import LearningWithNoisyLabels\n",
+    "\n",
+    "clf = LogisticRegression()\n",
```
Comment (jwmueller) [inline docs/source/notebooks/tabular_tutorial.ipynb:322]: In my newest version of tutorial I had replaced this line with:
`clf = LogisticRegression()  # Note we first re-initialize clf`

Can you update it to that? Thanks!

**Review by jwmueller — APPROVED** [review]
Already did my edits offline; just make the one line change I recommended in the comments below before we merge this.

File: docs/source/notebooks/tabular_tutorial.ipynb (hunk 1 above)
Comment (weijinglok) [reply docs/source/notebooks/tabular_tutorial.ipynb]: done!

**Review by jwmueller — APPROVED** [review]
already edited offline

**Pull request merged**